src/Tizen.NUI.Components/Controls/ItemsView/ItemsView.cs

```diff
@@ -170,93 +110,113 @@ public override void OnRelayout(Vector2 size, RelayoutContainer container)
-            //ItemsLayouter.RequestLayout(ScrollingDirection == Direction.Horizontal ? ContentContainer.CurrentPosition.X : ContentContainer.CurrentPosition.Y);
+
         }
 
-        private void DecorateItem(ViewItem item, int Index)
+        /// <summary>
+        /// </summary>
+        /// <param name="source"></param>
+        /// <param name="start"></param>
+        /// <param name="end"></param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void NotifyItemRangeChanged(IItemSource source, int start, int end)
         {
-            item.Index = Index;
-            item.ParentItemsView = this;
-            /* -- Template and BindinContext Need to be implemented in child class.
-            item.Template = (ItemsTemplate is DataTemplateSelector ?
-                             (ItemsTemplate as DataTemplateSelector).SelectDataTemplate(InternalItemSource.GetItem(Index), this) :
-                              ItemsTemplate);
-               item.BindingContext = InternalItemSource.GetItem(Index);
-            item.BindingContext = InternalItemSource.GetItem(Index);
-            */
-            item.Relayout += OnItemRelayout;
-            ContentContainer.Add(item);
-            /* -- This seems wrong as calling multiple times while scrolling
-            PropertyNotification noti = item.AddPropertyNotification("size", PropertyCondition.Step(0.1f));
-            noti.Notified += OnItemSizeChanged;
-            notifications.Add(noti);
-            */
+
+        }
+
+        /// <summary>
+        /// </summary>
+        /// <param name="source"></param>
+        /// <param name="startIndex"></param>
+        /// <param name="count"></param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void NotifyItemRangeInserted(IItemSource source, int startIndex, int count)
+        {
+
+        }
+
+        /// <summary>
+        /// </summary>
+        /// <param name="source"></param>
+        /// <param name="startIndex"></param>
+        /// <param name="count"></param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void NotifyItemRangeRemoved(IItemSource source, int startIndex, int count)
+        {
+
+        }
+
+        /// <summary>
+        /// </summary>
+        /// <param name="source"></param>
+        /// <param name="startIndex"></param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void NotifyItemRemoved(IItemSource source, int startIndex)
+        {
+
         }
 
         /// <summary>
         /// Realize indexed item.
         /// </summary>
         /// <param name="Index"> Index position of realizing item </param>
-        /// <since_tizen> 8 </since_tizen>
-        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
         [EditorBrowsable(EditorBrowsableState.Never)]
         internal virtual ViewItem RealizeItem(int Index)
```

Comment:
I have a minor suggestion that Index is changed to index.
https://www.c-sharpcorner.com/UploadFile/8a67c0/C-Sharp-coding-standards-and-naming-conventions/

Reply:
agreed. changes will be updated.